tests/test_monte_carlo.c

```diff
@@ -0,0 +1,152 @@
+#include <monte_carlo.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+void playAction(Board* board, Action* action) {
+  board->values[action->xPos * 3 + action->yPos] = action->player;
```

Comment:
I would put a define to specify why the number 3. It seems to be the size of the board, so I would write `#define BOARD_SIZE 3`